components/download_images/fondant_component.yaml

```diff
@@ -28,6 +28,10 @@ args:
     description: Number of times to retry downloading an image if it fails.
     type: int
     default: 0
+  n_connections:
+    description: Number of concurrent connections opened per process. Decrease this number if you are running into timeout errors.
```

Comment:
Can you also mention that lower connections improve the success rate but might result in slower downloads?